src/ts/src/NavBar.tsx

```diff
@@ -1,6 +1,5 @@
 import cx from "classnames";
 import React, { FunctionComponent } from "react";
-import { ReactComponent as AspenLogo } from "src/common/styles/logos/AspenLogo.svg";
```

Comment:
should we dump the svg file?

Reply:
Oh yes thanks for catching this! Done ✅ !